src/Sulu/Component/Util/SuluNodeHelper.php

```diff
@@ -70,7 +70,7 @@ public function getLanguagesForNode(NodeInterface $node)
         $languages = array();
         foreach ($node->getProperties() as $property) {
             /** @var PropertyInterface $property */
-            preg_match('/^' . $this->languageNamespace . ':([a-zA-Z_]*?)-title/', $property->getName(), $matches);
+            preg_match('/^' . $this->languageNamespace . ':([a-zA-Z_]*?)-changer/', $property->getName(), $matches);
```

Comment:
@danrot @wachterjohannes this is quite a bad bug .. the title is not always localized so this function returns bad information for some nodes. Using `created` doesn't work either, `changer` seems fine. In short, determining which locales a node has is a problem.